citrination_client/search/client.py

```diff
@@ -39,6 +41,12 @@ def pif_search(self, pif_system_returning_query):
         :return: :class:`PifSearchResult` object with the results of the query.
         :rtype: :class:`PifSearchResult`
         """
+
+        start_index = pif_system_returning_query.from_index or 0
+        size = pif_system_returning_query.size or 0
+        if start_index + size > MAX_QUERY_RESULTS:
+            raise CitrinationClientError("Citrination does not support pagination past the {}th result. Please change from_index and size arguments to be within this limit".format(MAX_QUERY_RESULTS))
```

Comment:
This validation happens the same way in two methods; I recommend that you extract it into a helper method so you don't have to repeat yourself.